relay-general/src/store/clock_drift.rs

```diff
@@ -0,0 +1,263 @@
+use std::fmt;
+
+use chrono::{DateTime, Duration as SignedDuration, Utc};
+
+use crate::processor::{ProcessValue, ProcessingState, Processor};
+use crate::protocol::Event;
+use crate::types::{Error, ErrorKind, Meta, ProcessingResult, Timestamp};
+
+/// The minimum clock drift for correction to apply.
```

Comment:
Would add the reasoning here or move the processor's docstring to the module-level

Reply:
This is already on the doc-string `ClockDriftProcessor`, which will be exported in a later PR (so that it can be used by non-processing Relays). 

@dashed you can find the reasoning in that same docstring:

```
/// There is a minimum clock drift of _55 minutes_ to compensate for network latency and small clock
/// drift on the sender's machine, but allow to detect timezone differences. For differences lower
/// than that, no correction is performed.
```

Let me know if you'd like me to clarify.